packages/react/src/Tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,160 @@
+.Tooltip {
+  display: none;
+}
+
+@media (forced-colors: active) {
+  .Tooltip {
+    outline: var(--brand-borderWidth-thin) solid transparent;
+  }
+}
+
+/* Overriding the default popover styles */
+.Tooltip[popover] {
+  --offset: var(
+    --base-size-8,
+    8px
+  ); /* This is a "magic number". Just gives some room to make sure the tooltip stay open when mouse leaves the trigger to hover tooltip */
+  /* these vars are for the triangle, via https://www.smashingmagazine.com/2024/03/modern-css-tooltips-speech-bubbles-part1/ */
+  --b: 1em; /* triangle base width */
+  --h: 0.5em; /* triangle height */
+  --p: 50%; /* center triangle position (0%:left 100%:right) */
+  --r: var(--brand-borderRadius-medium); /* border radius */
+  --c: var(--brand-color-neutral-emphasisPlus); /* bg color */
+
+  position: absolute;
+  padding: 0.5em 0.75em;
+  width: max-content;
+  margin: auto;
+  clip: auto;
+  white-space: normal;
+  font: normal normal var(--brand-text-size-100) / var(--brand-text-lineHeight-100) var(--brand-fontStack-sansSerif);
+  -webkit-font-smoothing: subpixel-antialiased;
+  color: var(--brand-color-text-onEmphasis);
```

Comment:
In dark mode this feels quite difficult to read, despite contrast ratio meeting requirements. Is it an aesthetic choice to use two dark values here? Or could we switch to using the default text token (white in dark mode) here?

Current:
<img width="332" alt="Screenshot 2024-04-16 at 09 51 11" src="https://github.com/primer/brand/assets/13340707/39396746-1c20-470a-89bd-15b6242bb342">



With white text:
<img width="374" alt=" " src="https://github.com/primer/brand/assets/13340707/ca891653-7db7-4470-bc51-53ebefbdb071">

Reply:
@rezrah - I'm comfortable with the white text in dark mode. Let me just confirm it meets the contrast requirement.